### ISSUE TITLE
Incompatibility with Node.js version 20.8.0

### ISSUE THREAD

**Issue by YonasValentin-aller** [plantagoIT/strapi-drag-drop-content-type-plugin#57]
## Environment

- **Package Version**: @retikolo/drag-drop-content-types@1.3.10
- **Node.js Version**: 20.8.0

## Description

I am encountering an issue when trying to install `@retikolo/drag-drop-content-types@1.3.10` using Node.js version 20.8.0. The installation fails due to the specified Node.js engine compatibility range, which does not include Node.js version 20.8.0.

## Steps to Reproduce

1. Run `yarn add @retikolo/drag-drop-content-types` with Node.js version 20.8.0 installed.
2. Observe the error indicating the incompatibility with the Node.js version.

## Expected Behavior

The package should install without any engine compatibility issues, or the documentation should clearly state the compatibility range to set proper expectations for users.

## Actual Behavior

The installation process fails with the following error message:

```
error @retikolo/drag-drop-content-types@1.3.10: The engine "node" is incompatible with this module. Expected version ">=14.19.1 <=18.x.x". Got "20.8.0"
error Found incompatible module.
```

**Comment by arastorhuiev** [comment]
same problem at latest strapi (4.15.1)

**Comment by derschiw** [comment]
Solved in 1.3.11
